### DISCUSSION TITLE
Step missing in install doc?

### DISCUSSION THREAD

**Discussion by jasonbrown1965** [LibreChat-AI/docs#75] [Q&A]
. . .

Following doc here for recommended docker install :
https://www.[librechat.ai/docs/local/docker](https://www.librechat.ai/docs/local/docker)

Git installed no problem, as did repository clone, change directory no worries, but step 3, namely 

**"cp .env.example .env"**

Returned an error:

**"cp: cannot stat '.env.example': No such file or directory"**

Tried copy too, but nah ... am guessing? There is a missing command to akshully create the file or directory first, before the cp or copy (Windows) command. Probably obvious for tech inclined, but confusing for non-experts like me.

**Comment by fuegovic** [comment]
make sure you're running the command from the cloned directory, the .env.example should be in there 
you can double check the directory content with the command `dir`
